docs/changelog/1_0_a7.rst

```diff
@@ -0,0 +1,218 @@
+    ...   module default {
+    ...     type Foo {
+    ...       property val -> str {
+    ...         default := 'n/a'
+    ...       }
+    ...     }
+    ...   }
+    ... };
+
+We use :eql:stmt:`DESCRIBE CURRENT MIGRATION AS JSON <DESCRIBE CURRENT
+MIGRATION>` to see what EdgeDB is proposing. The JSON format makes it
+easier to potentially integrate this with other tools. For this
+example it's worth turning on ``json`` output mode for edgedb REPL:
+
+.. code-block:: edgeql-repl
+
+    db> \set output-mode json
+    db[tx]> DESCRIBE CURRENT MIGRATION AS JSON;
+    [
+      {
+        "complete": false,
+        "confirmed": [],
+        "parent": "m16wif5skjyqd6dbp5uwa67qrgw422qcwa3vctx77z7r34yx5mbigq",
+        "proposed": {
+          "confidence": 1.0,
+          "operation_id": "CREATE TYPE default::Foo",
+          "prompt": "did you create object type 'default::Foo'?",
+          "statements": [{"text": "CREATE TYPE default::Foo {\n
+          CREATE OPTIONAL SINGLE PROPERTY val -> std::str {\n
+          SET default := 'n/a';\n    };\n};"}]
+        }
+      }
+    ]
+
+Since proposed statements look OK, we can go ahead and just apply the
+whole migration.
+
+.. code-block:: edgeql-repl
+
+    db[tx]> POPULATE MIGRATION;
+    OK: POPULATE MIGRATION
+    db[tx]> COMMIT MIGRATION;
+    OK: COMMIT MIGRATION
+
+Now, let's remove that ``default``, after all the property is optional.
+
+.. code-block:: edgeql-repl
+
+    db> START MIGRATION TO {
+    ...   module default {
+    ...     type Foo {
+    ...       property val -> str;
+    ...     }
+    ...   }
+    ... };
+    db[tx]> DESCRIBE CURRENT MIGRATION AS JSON;
+    [
+      {
+        "complete": false,
+        "confirmed": [],
+        "parent": "initial",
+        "proposed": {
+          "confidence": 0.9956623333333332,
+          "operation_id": "ALTER TYPE default::Foo",
+          "prompt": "did you alter object type 'default::Foo'?",
+          "statements": [{"text": "ALTER TYPE default::Foo {\n
+          ALTER PROPERTY val {\n        DROP default;\n    };\n};"}]
+        }
+      }
+    ]
+
+The proposed statements will ``DROP default`` for our property, so all
+seems to be in order and we can apply this migration, too, using
+:eql:stmt:`POPULATE MIGRATION` and :eql:stmt:`COMMIT MIGRATION`.
+
+We've also made improvements to the following migration features:
+
+* Better overall dependency tracking to make sure that migration to
+  the new state can be resolved and produces valid command sequence.
+* Type, index and alias renaming while keeping track of affected
+  expressions to make sure they don't become invalid (:eql:gh:`#1841`)
+* Function renaming (:eql:gh:`#1971`)
+* Moving a type between modules (:eql:gh:`#1890`).
+* Changing base types and changing where constraints are defined
+  (:eql:gh:`#1996`).
+
+
+Command-Line Tools
+==================
+
+* Deprecate ``--default-database`` and ``--default-database-user``
+  (:eql:gh:`#1879`).
```

Comment:
This is not `Command-line Tools` change, it's server command-line.

Reply:
OK

Reply:
@vpetrovykh Have you added what Paul suggested yet?

Reply:
Yes